helix-term/src/commands.rs

```diff
@@ -2566,7 +2568,7 @@ fn file_picker(cx: &mut Context) {
         cx.editor.set_error("Workspace directory does not exist");
         return;
     }
-    let picker = ui::file_picker(root, &cx.editor.config());
+    let picker = ui::file_picker(root, &cx.editor.config(), cx.editor.redraw_handle.0.clone());
```

Comment:
What do you think about adding a small `#[inline]` function to Editor to do the same as this? I'm thinking that if we refactor RedrawHandle in the future (for example from a tuple to a struct) we might end up having to change a lot of callsites. I'm not sure what a good name for this function would be.

Reply:
Yeah that's probably a good idea. An alternative I have been thinking about is to make the redraw handle a static. A `Notify` can be const initialized (and never needs mutable access) so this would avoid having to add the extra argument everywhere.